### PR TITLE
Development

### PR DESCRIPTION
**Pull request by mabalderramav**

### CODE REVIEW

**Comment by mabalderramav** [comment]
@carledriss ready to code review

**Comment by mabalderramav** [comment]
@carledriss ready to code review

File: src/test/java/org/fundacionjala/ocurrencestring/WildsTest.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test for {@link WildsTest}
+ */
+public class WildsTest {
+    final static String HAYSTACK = "Once upon a midnight dreary, while I pondered, weak and weary";
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/ocurrencestring/WildsTest.java]: this should be private

File: src/main/java/org/fundacionjala/ocurrencestring/SearchEngine.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ * Created by AldoBalderrama on 9/2/2016.
+ */
+public class SearchEngine {
+    final static String UNDERSCORE;
+    final static String BACKSLASH;
+    final static String BACKSLASH_Q;
+    final static String BACKSLASH_E;
+    final static String DOT;
+
+    static {
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/ocurrencestring/SearchEngine.java]: This is unnecessary for this case.

you can define directly like this.

``` java
private static final String UNDERSCORE = "_";
```

File: src/main/java/org/fundacionjala/ocurrencestring/SearchEngine.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        BACKSLASH_E = "\\E";
+        BACKSLASH_Q = "\\Q";
+        UNDERSCORE = "_";
+        BACKSLASH = "\\";
+        DOT = ".";
+    }
+
+    public static int find(String word, String text) {
+        String regWord = BACKSLASH_Q + word.replaceAll(UNDERSCORE,
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/ocurrencestring/SearchEngine.java]: This should be final
Try to use concat method instead of +

**Comment by carledriss** [comment]
@mabalderramav 
Let me know when you have fixed the observations

**Comment by mabalderramav** [comment]
@carledriss ready to code review

**Comment by carledriss** [comment]
Looks good to me!

**Pull request merged**